app/assets/stylesheets/layout/_profile.scss

```diff
@@ -4,27 +4,42 @@
 .profile {
   background: $color-ice-white;
   padding-top: 35px;
+  .panel {
+    border: 1px solid #b3b1a5;
+    border-radius: 0;
+  }
+  
   label {
     font-size: 20px;
-    font-weight: normal;
+    font-weight: lighter;
     color: $color-dark-grey;
   }
   .form-control {
     padding: 20px 26px;
-    font-size: 14px;
+    font-size: 20px;
+    font-weight: lighter;
     height: auto;
     color: $color-dark-grey;
-    background-color: $color-ice-white;
+    background: $color-ice-grey;
     background-image: none;
-    border: none;
+    border: 1px solid #b3b1a5;
     border-radius: 0;
-    box-shadow: none;
+    box-shadow: inset 4px 4px 9px rgba(0, 0, 0, 0.08);
     transition: none;
   }
-  .form-control:hover {
-    box-shadow: none;
-    transition: none;
+
+  .form-group {
+    padding-top: 15px;
+    padding-bottom: 15px;
   }
+
+  .account-data {
+    .panel-featured-block {
+      background: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets